EU4toV2/Source/V2World/V2World.cpp

```diff
@@ -1827,6 +1827,18 @@ void V2World::output() const
 	Utils::renameFolder("output/output", "output/" + theConfiguration.getOutputName());
 	createModFile();
 
+	// Record converter version
+
+	LOG(LogLevel::Debug) << "Writing version";
+	FILE* versionFile;

```

Comment:
Nit: I prefer C++ style files these days, both for the RAII aspects and because there's not the incompatibility with fopen_s.

Reply:
replaced with c++ ofstream handling.